packages/web/src/common/store/lineup/sagas.js

```diff
@@ -343,6 +345,27 @@ function* pause(action) {
   yield put(queueActions.pause({}))
 }
 
+function* togglePlay(lineupActions, lineupSelector, prefix, action) {
```

Comment:
this makes sense to me

Reply:
@sliptype lol rip i was doing the exact same thing in my branch 💀. but the good news is we both thought this was a good idea

Reply:
Oh noooo, sorry I should have communicated that